Extension/package.json

```diff
@@ -2553,8 +2730,122 @@
                                 "cert-pos47-c",
                                 "cert-sig30-c",
                                 "cert-str34-c",
+                                "Clang Static Analyzer core.*",
```

Comment:
Are these supposed to be added? Aren't these the same as the existing clang-analyzer-*?

Reply:
You tell me.  :)  They were listed.